### PR TITLE
add minAllowed field for KSM vpa

### PR DESCRIPTION
**Pull request by QuantumEnigmaa**
Towards https://github.com/giantswarm/giantswarm/issues/29538

### CODE REVIEW

**Comment by QuantumEnigmaa** [comment]
Maybe @giantswarm/team-turtles might want to review that as well :man_shrugging:

**Review by Gacko — APPROVED** [review]
LGTM!

**Comment by QuantumEnigmaa** [comment]
Done :)

**Review by Gacko — APPROVED** [review]

File: helm/kube-prometheus-stack/values.yaml [hunk 1]
```diff
@@ -330,6 +330,9 @@ kube-prometheus-stack:
       enabled: true
     verticalPodAutoscaler:
       enabled: true
+      minAllowed:
```
Comment (QuentinBisson) [inline helm/kube-prometheus-stack/values.yaml:333]: Can you add a comment to explain why we set this?

Comment (QuantumEnigmaa) [reply helm/kube-prometheus-stack/values.yaml:334]: Done

File: helm/kube-prometheus-stack/values.yaml [hunk 2]
```diff
@@ -330,6 +330,7 @@ kube-prometheus-stack:
       enabled: true
     verticalPodAutoscaler:
       enabled: true
+      # We set minAllowed=requests to allow VPA to work effiently
```
Comment (QuentinBisson) [inline helm/kube-prometheus-stack/values.yaml:333]:
```suggestion
      # We set minAllowed=requests to allow VPA to work efficiently
```

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**